core/src/main/java/com/google/cloud/sql/CredentialFactory.java

```diff
@@ -60,7 +61,7 @@ default GoogleCredentials getCredentials() {
           new AccessToken(
               credential.getAccessToken(),
               credential.getExpirationTimeMilliseconds() != null
-                  ? new Date(credential.getExpirationTimeMilliseconds())
+                  ? Date.from(Instant.ofEpochMilli(credential.getExpirationTimeMilliseconds()))
```

Comment:
No difference. `Date.from` returns a `Date` object. I made this change to avoid the warning. 

Another option is to keep `new Date` and add `@SuppressWarnings("JavaUtilDate")`. Should we go with this solution?

Reply:
Date.from seems fine in that case.